### DISCUSSION TITLE
test/layout/uncategorized/layout-comments/

### DISCUSSION THREAD

**Discussion by giscus** [SeanKilleen/seankilleen.github.io#601] [MinimalMistakesPR]
# test/layout/uncategorized/layout-comments/

This post should display comments.

http://127.0.0.1:4000/test/layout/uncategorized/layout-comments/

**Comment by SeanKilleen** [comment]
Testing

**Reply by SeanKilleen** [reply]
Reply

**Comment by SeanKilleen** [comment]
Ship it! :shipit:
